lib/__tests__/string-utils.test.ts

```diff
@@ -92,3 +93,23 @@ describe("allIsEmptyString", () => {
     },
   );
 });
+
+describe("isInternalLink", () => {
+  it.each`
+    input                                                                     | expected
+    ${"/faq"}                                                                 | ${true}
+    ${"https://www.wargabantuwarga.com"}                                      | ${true}
+    ${"https://www.wargabantuwarga.com/"}                                     | ${true}
+    ${"https://www.wargabantuwarga.com/provinces"}                            | ${true}
+    ${"https://www.wargabantuwarga.com/provinces"}                            | ${true}
+    ${"https://hotline.wargabantuwarga.com"}                                  | ${true}
```

Comment:
I think `https://hotline.wargabantuwarga.com` is not internal, it different subdomain.
It's safe to throw to new tab

Reply:
Updated. Let me know if it should be able to check link with no protocol (//www.wargabantuwarga.com), non http/https protocol, anchor tag  or other additional conditions.